### ISSUE TITLE
Alternative Font

### ISSUE THREAD

**Issue by jdittrich** [jdittrich/quickMockup#2]
The mockups should look provisional (useful for design communication and »politics«, since it says: »we may change it and it is far from finished«): So a »sketchy« font should be used, and Comic Sans is a good choice in this regard: It  looks hand drawn, is wide spread and readable.

However, Comic Sans is not Open Source, so if someone knows a font with the same metrics (means, if you change between this font and comic sans, the length of lines stays the same) we can use this other font as a preferred alternative. (As an addition, it would allow to skip annoying discussions about Comic Sans being inherently bad)

**Comment by simonv3** [comment]
Agreed with this. A Comic Sans-like font serves a good purpose of "this is temporary!"

**Comment by jdittrich** [comment]
Sadly, [Comic Neue](http://comicneue.com/) and [Purisa](http://packages.ubuntu.com/de/precise/fonts/fonts-tlwg-purisa) have rather different metrics than comic sans.
Most other options (TSCu_Comic etc.) I found have only one style and I suppose at least having a normal and a bold font would be good.

**Comment by simonv3** [comment]
Is it such a big deal if Comic Neue is narrower? Would we just re-align things?

**Comment by jdittrich** [comment]
If the designer used a narrow font and the user only has a wider one installed one may get unintended line breaks and thus twice-as-high elements and/or elements may not align properly anymore. So for now, I think it may do harm. 

But: If we would have some 10+ Mockups from different users we could just iterate through some fonts with all of them and see how bad it is and decide data-based :-)

**Comment by simonv3** [comment]
We could use @font-face to mitigate that? It adds some download to the page, but it would ensure that everyone sees the same font.

**Comment by jdittrich** [comment]
I'm sort of torn. In contrast to ComicSansMS, ComicNeue is
- Pro: Open Source Font 
- Contra: Not wide spread, almost anyone will need to install/download it  

But if we think it's worth it: 
- yes, that would be an idea – so if the user has the font installed, it just works, otherwise it is downloaded?
- And if yes, can we just add ComicNeue to the repo according to it's license?

**Comment by paulproteus** [comment]
FWIW I'm +0 to ​consistency in a tool like this.

Voting +0 not +1 since I'm just someone wandering by, but I also want to
say that I do love this project!
